src/eslint-config-adeira/ourRules.js

```diff
@@ -18,7 +18,7 @@ module.exports = {
   'no-compare-neg-zero': ERROR,
   'no-cond-assign': ERROR,
   'no-console': ERROR,
-  'no-constant-condition': [WARN, { checkLoops: false }],
+  'no-constant-condition': [ERROR, { checkLoops: false }],
```

Comment:
Can we make it a `NEXT_VERSION_ERROR` first? We usually don't jump from WARN to ERROR but give some time for the migration. 🤔

Reply:
ok